script.js

```diff
@@ -1,4 +1,68 @@
+var userWins = 0;
+var computerWins = 0;
+
+var numberGenerator = function () {
```

Comment:
Please start function names with verbs https://bootcamp.rocketacademy.co/course-logistics/naming-casing-and-commenting-conventions